Release 2025-05 (#993)
livingdocsIO/documentation: content/operations/releases/release-2025-05.md

This release addresses that gap by adding support for a broader range of metadata plugins in table dashboards, improving completeness and consistency across the system. Newly supported plugins include:

-	li-datetime, li-system-datetime
-	li-date, li-system-date
-	li-enum, li-system-enum
-	li-document-references
-	li-target-length

{{< feature-info "" "" >}}

### Additional oEmbed Providers :gift:

{{< feature-info "Clipboard" "editor" >}}

### Added Clipboard Context

This release enhances clipboard usability for nested components. Previously, when copying or cutting nested components, the clipboard displayed only the component name, with no descriptive context making it difficult to distinguish between components, especially those with identical names. The clipboard preview now includes the content of the first descendant as a description, providing better context and improving the overall editing experience.

{{< feature-info "Teaser Sidebar" "editor" >}}

### Clickable Titles in Teaser Sidebar

> Teaser titles are now clickable, allowing editors to directly open the linked article in a new window without needing to navigate to it manually. This enables more efficient workflows for all those roles that work with the teaser sidebar.

Reviewer: Just asked Sharon, she said it can go in. Next time we can cover all the small UI changes in a UI polish section or something like that, to make it more cohesive. :)

### Media Center - Delete language metadata set :gift:

{{< feature-info "Table Dashboards" "editor" >}}

### Table Dashboards - Support All Metadata Cells :gift:

> In earlier iterations of the table dashboards, support for metadata plugins was intentionally limited to reduce initial implementation complexity. However, as the feature matured, the lack of support for certain plugins introduced inconsistencies and additional coordination overhead when extending functionality on demand.

Reviewer: ```suggestion
In earlier iterations of the Table Dashboards, metadata plugin support was limited. However, as the feature has become more widely adopted inconsistencies across plugin behaviour became confusing and problematic.
```

I wouldn't mention reduce initial implementation complexity. It somehow devalues the efforts we make to bring features to users.

This release addresses that gap by adding support for a broader range of metadata plugins in table dashboards, improving completeness and consistency across the system. Newly supported plugins include:

-	li-datetime, li-system-datetime
-	li-date, li-system-date
-	li-enum, li-system-enum
-	li-document-references
-	li-target-length

{{< feature-info "" "" >}}

### Additional oEmbed Providers :gift:

{{< feature-info "Clipboard" "editor" >}}

### Added Clipboard Context

> A small improvement in this release enhances clipboard usability for nested components.

Reviewer: ```suggestion
This release enhances clipboard usability for nested components. 
```

Previously, when copying or cutting nested components, the clipboard displayed only the component name, with no descriptive context making it difficult to distinguish between components, especially those with identical names.

> The clipboard preview now includes the content of the first descendant as a description, providing better context and improving the overall editing experience.

Reviewer: ```suggestion
The clipboard preview now includes the content of the first child as a description, providing better context and improving the overall editing experience.
```
I am not sure if the right word is `child` or `descendant`. `descendant` sound a lot like God given.

{{< feature-info "Teaser Sidebar" "editor" >}}

### Clickable Titles in Teaser Sidebar

> Another small improvement in this release makes working with the teaser sidebar more efficient. Teaser titles are now clickable, allowing editors to directly open the linked article in a new window without needing to navigate to it manually.

Reviewer: ```suggestion
Teaser titles are now clickable, allowing editors to directly open the linked article in a new window without needing to navigate to it manually. This enables more efficient workflows for all those roles that work with the teaser sidebar companion.
```

Author: I like the change but couldn't we just leave out companion at the end?

Reviewer: I felt it needed some AI spin.
